rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/data/DataUtility.java

```diff
@@ -299,4 +301,14 @@ public static TaskId taskId(JobId jobId, TaskIdData taskIdData) {
         return createTaskId(jobId, taskIdData.getReadableName(), taskIdData.getId());
     }
 
+    public static JobLabelInfo toJobLabelInfo(JobLabelInfoData jobLabel) {
+        return new JobLabelInfo(jobLabel.getId(), jobLabel.getLabel());
+    }
+
+    public static List<JobLabelInfo> toJobLabelsInfo(List<JobLabelInfoData> jobLabel) {
+        List<JobLabelInfo> jobLabelInfo = new LinkedList<>();
```

Comment:
better to use an ArrayList with the same size as the input.

List<JobLabelInfo> jobLabelInfo = new ArrayList<>(jobLabel.size());